examples/nemo/eg5/README.md

```diff
@@ -0,0 +1,91 @@
+# PSyclone NEMO Example 5 - Kernel Data Extraction
+
+**Author:** J. Henrichs, Bureau of Meteorology
+
+
+This example demonstrates the use of PSyclone to add code for kernel
+extraction, i.e. writing input- and output-data of a kernel into a data file.
+
+Once you have installed PSyclone, either script may be supplied to
+PSyclone via the -s option, e.g.:
+
+```sh
+psyclone -nodm -l all -api nemo -s ./extract_kernels.py ../code/tra_adv.F90
+```
+
+Executing this will output the transformed Fortran code with the 
+kernel extraction code added. Note that some of the lines in this
+Fortran code will exceed the 132-character limit. This may be remedied
+by supplying the `-l all` flag to PSyclone (as is done in the Makefile).
+Kernel extraction does not support distributed memory (TODO #1992), so
```

Comment:
As in the Makefile, the NEMO API doesn't have explicit support DM so you don't need '-nodm'.

Reply:
Removed.